test/torchaudio_unittest/functional/batch_consistency_test.py

```diff
@@ -216,28 +209,20 @@ def test_compute_kaldi_pitch(self):
         n_channels = 2
         waveform = common_utils.get_whitenoise(sample_rate=sample_rate, n_channels=self.batch_size * n_channels)
         batch = waveform.view(self.batch_size, n_channels, waveform.size(-1))
-        self.assert_batch_consistency(F.compute_kaldi_pitch, batch, sample_rate=sample_rate)
+        self.assert_batch_consistency(F.compute_kaldi_pitch, (batch,), sample_rate=sample_rate)
 
     def test_lfilter(self):
         signal_length = 2048
         torch.manual_seed(2434)
         x = torch.randn(self.batch_size, signal_length)
         a = torch.rand(self.batch_size, 3)
         b = torch.rand(self.batch_size, 3)
-
-        batchwise_output = F.lfilter(x, a, b, batching=True)
-        itemwise_output = torch.stack([F.lfilter(x[i], a[i], b[i]) for i in range(self.batch_size)])
-
-        self.assertEqual(batchwise_output, itemwise_output)
```

Comment:
I just realized that this method was not honoring the `self.dtype` and `self.device` configuration. Grad that you pulled up this PR.

Reply:
There're other methods that use `torch.rand` without `self.dtype` and `self.device`, shall we also update them in this PR?

Reply:
If they use `self.assert_batch_consistency`, it's fine, otherwise we need to fix it. (but let's do it in a separate PR.)